tools/acorn-optimizer.js

```diff
@@ -26,6 +26,15 @@ function assert(condition, text) {
   }
 }
 
+function assertAt(condition, offset, message) {
```

Comment:
```suggestion
function assertAt(condition, offset, message = '') {
```
(without this, won't not passing it in cause an "undefined" string to appear?